br/pkg/lightning/restore/precheck_test.go

```diff
@@ -0,0 +1,40 @@
+package restore
```

Comment:
ditto

Reply:
with goland, you can set a `go file template`